Lib/test/_test_multiprocessing.py

```diff
@@ -3773,12 +3773,16 @@ def _attach_existing_shmem_then_write(shmem_name_or_obj, binary_data):
         local_sms.buf[:len(binary_data)] = binary_data
         local_sms.close()
 
+    def _new_shm_name(self, prefix):
```

Comment:
Would you mind to add a comment mentioning "bpo-45310" which explains why it adds a pid as a suffix? Just explain that it's to run tests in parallel.

Reply:
```suggestion
    def _new_shm_name(self, prefix):
        # Add a PID to the name of a POSIX shared memory object to allow
        # running multiprocessing tests (test_multiprocessing_fork,
        # test_multiprocessing_spawn, etc) in parallel.
```